src/db/models.ts

```diff
@@ -31,7 +31,7 @@ export interface URLAttributes {
 
 export const URLs = db.define<URLAttributes, URLAttributes>('url', {
   code: {
-    type: Sequelize.INTEGER,
+    type: Sequelize.BIGINT,
```

Comment:
@championswimmer When creating codes for characters of length 7-9 integers where greater than 2147483647 and hence I guess we'll need BIGINT

Reply:
+1